src/solver/conv_MP_bidirectional_winograd.cpp

```diff
@@ -87,6 +98,11 @@ MIOPEN_DECLARE_ENV_VAR(MIOPEN_CONV_PRECISE_ROCBLAS_TIMING)
     GenerateClangDefsym((options), "fdilation_w", params.kernel_stride_w);                         \
     GenerateClangDefsym((options), "fdilation_h", params.kernel_stride_h);
 
+#define DEFINE_WORKSPACE_OFFSETS(in_size, wei_size, out_size) \
+    size_t wino_in_offset  = 0;                               \
+    size_t wino_out_offset = (in_size);                       \
+    size_t wino_wei_offset = wino_out_offset + (out_size);
```

Comment:
```suggestion
struct WinoOffsets {
    const size_t in, out, wei;
    WinoOffsets(size_t in_size, size_t wei_size, size_t out_size)
    : in(0), out(in_size), wei(in_size + out_size)
    {}
};
```
and so on. Make it const as much as possible.

Reply:
Make it const or make it struct???

Reply:
Make it struct first. Then you can replace macro invocation with
```C++
const WinoOffsets winoOffset(in_size, wei_size, out_size);
...
// use:
... winoOffset.in ...
```